Highway.h

```diff
@@ -2,6 +2,7 @@
 
 #include <vector>
 #include <algorithm>
+#include <string>
```

Comment:
You are not using `<string>` in this header file.  
Don't include what you don't use.
- [ ] revise.